.github/workflows/build-push-tool-images.yml

```diff
@@ -0,0 +1,66 @@
+name: Build & Push Tool Image to GHCR
```

Comment:
I was thinking we would only trigger image builds as needed, not on every PR. Most PRs they probably won't change

Reply:
I did add `path`. Only one of the tool Dockerfile changes, this will be triggered.

Reply:
That means if one gets rebuilt all get rebuilt right? I was thinking we'd just have a button per tool or something

Reply:
you mean having GHA figure out which Docker file is changed and just trigger that tool build?